azure-mgmt-compute/src/test/java/com/azure/management/compute/VirtualMachineAvailabilityZoneOperationsTests.java

```diff
@@ -48,16 +50,41 @@ protected void cleanUpResources() {
     @Test
     public void canCreateZonedVirtualMachineWithImplicitZoneForRelatedResources() throws Exception {
         final String pipDnsLabel = generateRandomResourceName("pip", 10);
+        final String nsgName = generateRandomResourceName("nsg", 10);
+        final String nicName = generateRandomResourceName("nic", 10);
         final String proxyGroupName = "plg1Test";
+        //Create resource group
+        ResourceGroup resourceGroup = resourceManager.resourceGroups().define(RG_NAME)
+                .withRegion(REGION)
+                .create();
+        //Create network security group
+        NetworkSecurityGroup nsg = networkManager.networkSecurityGroups().define(nsgName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroup)
+                .create();
+        //Create public IP address
+        PublicIPAddress pip = networkManager.publicIPAddresses().define(pipDnsLabel)
```

Comment:
Please keep the test case aka. example back compatible. Not sure why so much changes for this example?

Reply:
@xccc-msft You might want to double check with Tanyi. After SdkContext move to manager, and code use this SdkContext for random naming for inline resource creation, at least my test runs good on appservice (which previously got 2 kind of inline, StorageAccount, and AppServicePlan).